docs/UsersGuide/source/Reference/Glossary.rst

```diff
@@ -102,6 +102,9 @@ Glossary
    GFS
       `Global Forecast System <https://www.ncei.noaa.gov/products/weather-climate-models/global-forecast>`_. The GFS is a National Centers for Environmental Prediction (:term:`NCEP`) weather forecast model that generates data for dozens of atmospheric and land-soil variables, including temperatures, winds, precipitation, soil moisture, and atmospheric ozone concentration. The system couples four separate models (atmosphere, ocean, land/soil, and sea ice) that work together to accurately depict weather conditions.
 
+   GOCART
+      NASA's Goddard Chemistry Aerosol Radiation and Transport (GOCART) model simulates the distribution of major tropospheric aerosol types, including sulfate, dust, organic carbon (OC), black carbon (BC), and sea salt aerosols. The UFS Weather Model integrates a prognostic aerosol component using GOCART. The code is publicly available on GitHub at https://github.com/GEOS-ESM/GOCART. 
+
    GRIB2 
       The second version of the World Meterological Organization's (WMO) standard for distributing gridded data.  
 
```

Comment:
Should the GSI section below (lines 111 and 112) be removed, or should these remain until the GSI has been fully removed from the SRW App?

Reply:
Yes, good catch! :)